ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorWidget.java

```diff
@@ -188,6 +194,7 @@ public OrionEditorWidget(final ModuleHolder moduleHolder,
         initWidget(UIBINDER.createAndBindUi(this));
 
         this.uiUtilsOverlay = moduleHolder.getModule("UiUtils");
+        this.annotationType = moduleHolder.getModule("OrionAnnotations").<OrionAnnotationsOverlay>cast().getAnnotationType();
```

Comment:
maybe 
`this.annotationType = ((OrionAnnotationsOverlay)moduleHolder.getModule("OrionAnnotations")).getAnnotationType(); `
more readable?